pkg/controller/migcluster/remote_watch_map.go

```diff
@@ -0,0 +1,75 @@
+/*
```

Comment:
I appreciate the effort to be modular here and put the `Map` in a separate module.  However, seems reasonable for all things _remote watching_ could just to in `remotewatch.go.`

Reply:
ACK, originally had some of this logic in the `util` package and was running into the compiler not wanting me to express a circular dependence from `migcluster <-> util`, and think I moved `remote_watch_map.go` over at some point. 

Agree it can be integrated into the same file now.